web/pdf_page_view.js

```diff
@@ -52,6 +52,85 @@ import { TextHighlighter } from "./text_highlighter.js";
 import { TextLayerBuilder } from "./text_layer_builder.js";
 import { XfaLayerBuilder } from "./xfa_layer_builder.js";
 
+// TODO: Should we make this an app option, or just always
+// enable the feature?
+const ENABLE_ZOOM_DETAIL = true;
```

Comment:
Making this an `AppOption` seems like a good idea to me, since that may help future testing/debugging, however it probably does *not* need it to be a preference.
(In any case, leaving it as a code constant seems "wrong".)